recipes/veloxchem/meta.yaml

```
@@ -0,0 +1,113 @@
+{% set name = "veloxchem" %}
+{% set version = "1.0" %}
+{% set pre = "rc3" %}
+{% set sha256 = "ed22aeb72d832f0daccffa393a1a349dab65fc85cefbb6a069933d3b51ad7ffd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/veloxchem/veloxchem/-/archive/v{{ version }}-{{ pre }}/veloxchem-v{{ version }}-{{ pre }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/geometric-as-posix.patch
+
+build:
+  number: 0
+  script_env:
+    - OMPI_MCA_plm_rsh_agent=  # [mpi == 'openmpi' and linux]
+  binary_relocation: true
+  entry_points:
+    - vlx=veloxchem.main:main
+  ignore_run_exports:
+    - python
+  run_exports:
+    - {{ pin_subpackage('veloxchem', max_pin='x.x') }}
+
+requirements:
+  build:
+    - cmake >=3.18
+    - {{ compiler('cxx') }}
+    - libgomp      # [linux]
+    - llvm-openmp  # [osx or win]
+    - ninja
+    - pkg-config   # [unix]
+    - gtest
+    - gmock
+    - pybind11 >=2.6,<3.0
+  host:
+    - python
+    - pybind11-abi
+    - {{ mpi }}
+    - mpi4py >=3.0
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
```

```suggestion
    - libcblas
    - liblapacke
```
[The link inspector thinks](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=610068&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=49424) that libblas and liblapack are unused. Static linking is not allowed.

---

Fixed.